include/nbl/builtin/hlsl/math/constants.hlsl

```diff
@@ -0,0 +1,23 @@
+
+// Copyright (C) 2018-2022 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_BUILTIN_HLSL_MATH_CONSTANTS_INCLUDED_
+#define _NBL_BUILTIN_HLSL_MATH_CONSTANTS_INCLUDED_
+
+namespace nbl
+{
+namespace hlsl
+{
+namespace math
+{
+	static const float PI = 3.14159265359;
+	static const float RECIPROCAL_PI = 0.318309886183;
+	static const float SQRT_RECIPROCAL_PI = 0.56418958354;
+}
+}
+}
```

Comment:
needs to be unified with others which I mentioned with the proposal 0003 from HLSL